subscription-manager.spec

```diff
@@ -177,8 +177,10 @@ Requires: %{name} = %{version}-%{release}
 %if %use_gtk3
 Requires: pygobject3
 Requires: gtk3
+Requires: abattis-cantarell-fonts
 %else
 Requires: pygtk2 pygtk2-libglade
+Requires: dejavu-sans-fonts
```

Comment:
> `font(dejavusans)`

To be clear, this will work for RHEL6, but not SLES11.

So, explicitly, the other option is:

```specfile
%if 0%{?suse_version}
Requires: dejavu
%else
Requires: font(dejavusans)
%endif
```

but I guess looking at it now, it's uglier than just enumerating both the dejavu packages explicitly. So, I'd prefer the first dejavu example I gave.